tests/onepay/test_transaction.py

```diff
@@ -39,6 +39,9 @@ def test_create_options(self):
         self.assertEqual(options.shared_secret, "shared_secret")
 
     def test_validate_create(self):
+        onepay.callback_url = None
```

Comment:
If you only make this change to the test and NOT apply the fix in transaction.py, do the test fail?

Reply:
Also, if that's the case: Are there any other test that might be wrong because they didn't reset the static values? Should that go into the `setup()` of many tests?

Reply:
> If you only make this change to the test and NOT apply the fix in transaction.py, do the test fail?

Yes, it fails.

> Also, if that's the case: Are there any other test that might be wrong because they didn't reset the static values? Should that go into the `setup()` of many tests?

Both static values aren't required nor used in other tests, anyway right now I moved both static values to `setUp()` because are part of the initialization.